examples/jwt/src/backend/app/core.clj

```diff
@@ -0,0 +1,43 @@
+(ns app.core
+  (:require
+    [app.config :refer [init-jwt-cfg]]
+    [app.controllers.index :as index]
+    [app.controllers.login :as login]
+    [app.controllers.secret :as secret]
+    [piotr-yuxuan.closeable-map :refer [closeable-map]]
+    [xiana.config :as config]
+    [xiana.handler :as x-handler]
+    [xiana.interceptor :as x-interceptors]
+    [xiana.interceptor.error]
+    [xiana.jwt.interceptors :as jwt-interceptors]
+    [xiana.route :as x-routes]
+    [xiana.webserver :as ws]))
+
+(def routes
+  [["" {:handler x-handler/handler-fn}]
+   ["/" {:action index/index}]
+   ["/login" {:post
+              {:action       login/login-controller
+               :interceptors {:except [jwt-interceptors/jwt-auth]}}}]
+   ["/secret" {:post
+               {:action secret/protected-controller}}]])
+
+(defn ->system
+  [app-cfg]
+  (-> (config/config)
+      init-jwt-cfg
+      (merge app-cfg)
```

Comment:
the (config/config) merging its parameter if I'm right. It would be more elegant to pass the app-cfg to it